src/connection/secret_connection/amino_types.rs

```diff
@@ -0,0 +1,15 @@
+//! Amino types used by Secret Connection
+
+use prost_amino_derive::Message;
+
+/// Authentication signature message
+#[derive(Clone, PartialEq, Message)]
+pub struct AuthSigMessage {
```

Comment:
I guess this is no longer in `tendermint-rs`? /cc @ebuchman @Liamsi

Reply:
Nope, it's SecretConnection specific